Makefile `example-csv` command fail

```
 ~/src/github.com/zhenik/data-mesh │ master *2 !1 ?2  make example-csv                                                                                                                                            2 х │ 20s │ 14:06:12 
NOMAD_ADDR=http://172.17.0.1:4646 nomad stop -purge example-csv | true
No job(s) with prefix or id "example-csv" found
sleep 2
NOMAD_ADDR=http://172.17.0.1:4646 nomad run nomad-jobs/example-csv.hcl
Error submitting job: Unexpected response code: 500 (1 error occurred:
        * Task group test validation failed: 1 error occurred:
        * Task group service validation failed: 1 error occurred:
        * Check testdata-available-via-beeline invalid: refers to non-existent task beeline





)
Makefile:187: recipe for target 'example-csv' failed
make: *** [example-csv] Error 1
```

--- a/nomad-jobs/example-csv.hcl
+++ b/nomad-jobs/example-csv.hcl
@@ -23,7 +23,7 @@ job "example-csv" {
         }
       }
       check {
-        name = "beeline"
+        name = "hiveserver-available-via-beeline"
         type = "script"
         task = "beeline"
         command = "/bin/bash"

